packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -238,6 +238,8 @@ export class CdkToolkit {
       if (requireApproval !== RequireApproval.Never) {
         const currentTemplate = await this.props.deployments.readCurrentTemplate(stack);
         if (printSecurityDiff(currentTemplate, stack, requireApproval)) {
+          // Lock the logger from logging temporarily
+          setLogLock(true);
```

Comment:
This feature is uncommonly (but amusingly) called "corking" [one](https://www.educative.io/answers/what-is-stream-module-cork-in-nodejs) [two](https://lists.freedesktop.org/archives/pulseaudio-discuss/2009-December/005767.html).

Can we call it `corkLogging()` and `uncorkLogging()` ?